src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function creatPromise() {
+  const resolver = (resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+
+    setTimeout(() => {
+      reject(new Error());
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+const promise1 = creatPromise();
+const promise2 = creatPromise();
```

Comment:
there should be 2 different promises
and working separately 

> promise1 should be resolved on a click on .logo.
promise2 promise should be rejected in 3 seconds.

Reply:
fixed